symengine/sympy_compat.py

```diff
@@ -126,6 +126,19 @@ def __new__(cls, a):
         return symengine.gamma(a)
 
 
+class Derivative(_RegisteredFunction):
+    _classes = (symengine.Derivative,)
+
+    def __new__(cls, expr, variables):
+        return symengine.diff(expr, variables)
+
+
+class Subs(_RegisteredFunction):
+    _classes = (symengine.Subs,)
+
+    def __new__(cls, *args):
+        return symengine.Subs.subs(*args)
```

Comment:
@isuruf How should this call be modified, as there isn't any standalone `subs` function in the wrapper?